.github/workflows/jira-issue-create.yml

```diff
@@ -18,19 +36,29 @@ jobs:
           JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
           JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
 
+      - name: Convert Label String
+        id: string
+        uses: ASzc/change-string-case-action@v1
+        with:
+          string: ${{ inputs.label }}
```

Comment:
How does this work for TS?

Reply:
Good point. 
Mark the label default as TS and not required.

Reply:
I wonder if we can create this usable workflow in ts but also have something like https://github.com/yuhao900914/Amplitude-Flutter-Example/blob/main/.github/workflows/main.yml in ts

Reply:
That's a good point. I think in this case the label is always required, then we will not miss or assign the wrong label. I will try this on my end.